electron/js/menu/TrayIconHandler.js

```diff
@@ -0,0 +1,85 @@
+const {app, Menu, nativeImage, Tray} = require('electron');
+const config = require('./../config');
+const environment = require('./../environment');
+const lifecycle = require('./../lifecycle');
+const locale = require('./../../locale/locale');
+const path = require('path');
+const windowManager = require('./../window-manager');
+
+class TrayIconHandler {
+  constructor(appIcon = new Tray(nativeImage.createEmpty())) {
+    this.appIcon = appIcon;
+    this.icons = this.createIcons();
+    this.lastUnreadCount = 0;
+    this.initTrayIcon();
+  }
+
+  initTrayIcon() {
```

Comment:
No need to expose this to the public

Reply:
Because we don't really have a way to create private method and we might not want to be fancy and play with `call`, `apply` or `bind` a solution could be just to inline this function in the constructor.
It's a pretty small function, so I think it's fine to inline